lua-docs/content/reference/shape.yml

```diff
@@ -16,6 +16,15 @@ constructors:
 
 properties:
 
+    - name: "Center"
+      type: "Number3"
+      description: |
+          Provides the coordinates of the [Block] at the center of the [This].
```

Comment:
This is not true. Could be half a block for example if the shape has an uneven number of cubes on width, height or depth. Center is going to be a shortcut to `shape.BoundingBox.Center`.